msteams-platform/bots/how-to/conversations/bot-copilot-handoff.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Copilot handoff
+description: Learn how to allow users to continue their conversation with custom copilots from the Copilot for Microsoft chat.
+ms.date: 05/07/2024
+ms.topic: conceptual
+author: v-ypalikila
+ms.author: surbhigupta
+---
+
+# Copilot handoff
+
+You can enable Copilot handoff to allow users to continue the conversation with a custom Copilot from the Copilot for Microsoft chat without losing the context of their work. You can improve your plugins with the Copilot handoff feature, which allows for a seamless transition to your custom copilot by using deep links that carry over the user context. This resolves the issue of users having to restart and repeat their requests in the bot experience. For example, in the following image, the user is using Copilot to get information on tech issues and receives results from across the enterprise dataset. The user doesn't find the information sufficient and wants to continue an interaction with an Aisera bot.​ When the user selects the Aisera bot, a new chat is opened to continue the conversation with Aisera. The continuation of the conversation without losing the context is called Copilot handoff.
```

Comment:
Please break up and simplify this wall of text, as it is hard to read.

Reply:
Updated the Introduction and split the intro and examples into two paragraphs.